packages/devtools_server/lib/src/server.dart

```diff
@@ -217,7 +217,7 @@ Future<void> registerLaunchDevToolsService(
         // quietly replace the IP with "penguin.linux.test". This is not valid
         // for us since the server isn't bound to the containers IP (it's bound
         // to the containers loopback IP).
-        path: devToolsUri.path ?? '/',
+        path: devToolsUri.path == '' ? '/' : devToolsUri.path,
```

Comment:
nit: 
use .isEmpty instead of == ''

Reply:
Oops, I always forget about that on strings! Opened https://github.com/flutter/devtools/pull/610 since this had already landed.